### ISSUE TITLE
Missing Validation message on Blank User/Email and Password Fields

### ISSUE THREAD

**Issue by namrata-modus** [ModusCreateOrg/beep#98]
**Steps to Reproduce**
1. Open server/Ios App/Android App
2. Click on "Account" icon
3. Keep "Username& Email"/"Password" field blank
4. Click on "Check" link 

Result: No 'Validation' appears  for "Username OR email" OR "Password" Fields

**Comment by michaeltintiuc** [comment]
We might need designs from @tasteofdalife

**Comment by tasteofdalife** [comment]
@michaeltintiuc is this still opened? if so, what possible errors (error texts) could we have, accept Blank fields? I though if the input field is empty, the buttons are disabled, thus, no need to show Validation Errors?

**Comment by divz11** [comment]
Yes I think there is no need to show warning as check button is disabled if input is empty
